src/main/resources/application.yaml

```diff
@@ -55,3 +55,5 @@ scheduling:
     delete-dispatched-files:
       enabled: true
       cron: ${DELETE_DISPATCHED_FILES_CRON}
+    scan:
+      delay: ${TASK_SCAN_DELAY:30000} # In milliseconds
```

Comment:
This PR won't work unless we have this in our charts. I assume you are creating a separate PR ?

Reply:
sorry missed that it has default. As discussed let's remove the default one once we have the chart PR